src/ec/packages/ec-component-text-input/__snapshots__/text-input.test.js.snap

```diff
@@ -108,26 +116,28 @@ exports[`EC - Text field Disabled renders correctly with extra attributes 1`] =
   class="ecl-form-group ecl-form-group--text-input"
 >
   <label
-    class="ecl-form-label"
+    class="ecl-form-label ecl-form-label--disabled"
     for="example-id"
   >
-    Label
+    Label        
+    <span
+      class="ecl-form-label__optional"
+    />
   </label>
+  <div
+    class="ecl-help-block  ecl-help-block--disabled"
+  >
+    Help message
+  </div>
   <input
-    class="ecl-text-input"
+    class="ecl-text-input ecl-text-area--m"
```

Comment:
yep, the test is revealing the issue about the class

Reply:
updated.